### PR TITLE
Adding qliot.py for ambient light.

### PR DESCRIPTION
**Pull request by lsabalo**

### CODE REVIEW

File: minqlx/qliot.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+INTERESTING_GAME_TYPES = ['ad', 'ctf']
+
+def connected(client):
+  print('Connected to Adafruit IO! Listening for updates...')
+  client.subscribe('qliot')
+
+def disconnected(client):
+  print('Disconnected from Adafruit IO!')
+  #sys.exit(1)
```
Comment (elindner) [inline minqlx/qliot.py]: remove this

File: minqlx/qliot.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+def message(client, feed_id, payload):
```
Comment (elindner) [inline minqlx/qliot.py]: remove this function, it's empty

File: minqlx/qliot.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  pass
+
+class qliot(minqlx.Plugin):
+  def __init__(self):
+    self.add_hook('player_loaded', self.handle_player_loaded)
+    self.add_hook('player_disconnect', self.handle_player_disconnect)
+    self.client = MQTTClient(ADAFRUIT_IO_USERNAME, ADAFRUIT_IO_KEY)
+    self.client.on_connect = connected
+    self.client.on_disconnect = disconnected
+    self.client.on_message = message
```
Comment (elindner) [inline minqlx/qliot.py]: remove this, the function doesn't do anything

**Comment by lsabalo** [comment]
Changes made per request

**Pull request merged**